test/bats/oathkeeper.bats

```diff
@@ -88,7 +88,7 @@ exec_oathkeeper() {
 
 oathkeeper_jwt() {
   echo $output \
-  | grep -oP 'Authorization: Bearer \K[^ ]+'
+  | echo $output | grep -o 'Authorization: Bearer [^ ]*' | sed 's/Authorization: Bearer //'
```

Comment:
The `echo $output` here is likely redundant?
```suggestion
  | grep -o 'Authorization: Bearer [^ ]*' \
  | sed 's/Authorization: Bearer //'
```

Reply:
> I'm ok with sticking with sed as well if we prefer that
no preference on my end. code provide by GPT :D